playbooks/roles/bats/tasks/main.yml

```diff
@@ -22,6 +22,7 @@
     update: "{{ bats_update_forklift }}"
 
 - name: "Run bats"
-  shell: "bats {{ bats_tests }}"
+  shell: "bats {{ item }} 2>&1 | tee /root/{{ item }}.out"
```

Comment:
Also, what's the rationale for ```.out```? Since they're in tap output, would ```.tap``` be a logical choice?